www/%team/distributing/%to.json.spt

```diff
@@ -9,87 +9,77 @@ from gratipay.models.participant import Participant
 from gratipay.models.team.mixins.takes import ZERO, PENNY
 
 [--------------------]
-request.allow('GET', 'POST')
+request.allow('POST')
+
 team = get_team(state)
 if team.available == 0:
     website.redirect('..', base_url='')
 
-member = Participant.from_id(request.path['member_id'])
-if member is None or member.is_suspicious:
+if user.ANON:
+    raise Response(401)
+recorder = user.participant
+
+participant = Participant.from_id(request.path['to'])
+if participant is None:
     raise Response(404, _("User not found."))
-is_member = member.member_of(team)
 
-if request.method == 'POST':
+participant_is_member = participant.member_of(team)
+recorder_is_owner = team.owner == recorder.username
 
-    if user.participant not in (member, team):
-        raise Response(403)
+if request.method == 'POST':
```

Comment:
Don't we already have https://github.com/gratipay/gratipay.com/pull/4077/commits/a7c5fae31432896ca3597310facf79e8e69e8aff#diff-0f0c9aee41837b89ae6ba33b2e09965bR12 ?